XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityController.cs

```diff
@@ -37,6 +37,18 @@ public interface IMixedRealityController
         /// </summary>
         IMixedRealityControllerVisualizer Visualizer { get; }
 
+        /// <summary>
+        /// The controller's angular velocity, if any.
+        /// Angular velocity refers to how fast an object rotates or revolves relative
+        /// to another point.
+        /// </summary>
+        UnityEngine.Vector3 AngularVelocity { get; }
```

Comment:
`IMixedRealityController` has two new properties for velocity and angular velcotiy added. But it seems they are only required for hand tracking currently. If other controller's can't profit here, it would make sense to introduce e.g. `IMixedRealityHandController` to derive from `IMixedRealityControlle`

Reply:
We should include these in the interface, even VR systems expose this (just not WMR).  SO I agree to add these in to the interface and base class